internal/controller/telemetry/otel_col_conf_test_fixtures/complex.yaml

```diff
@@ -155,13 +155,35 @@ connectors:
                       default_value: no_subscription
                     - key: exporter
                       default_value: no_exporter
+                resource_attributes:
+                    - key: k8s.namespace.name
+                      default_value: no_namespace
+                    - key: k8s.node.name
```

Comment:
What will be the name of the label in the metric? Can we override the name for it?

Reply:
`k8s_node_name`, this is parsed from the attribute name by prometheusexporter. Of course we can rename it in the metrics pipeline while it's an attribute/resource attribute